CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added `cc-password` feature in `Cargo.toml`.
 - Applied Clumit theme to `Sort`, `SelectMini`, `SelectSearchable`, `Input`,
   `view_asterisk`, `Notification`.
+- Applied Clumit theme to `WholeList`, `Tag`, and `HostNetworkHtml`.
```

Comment:
This should be include in the section "Unreleased"

Reply:
Since @jinjoolee07 is out of office for a long time, I moved the entry in the CHANGELOG to the Unreleased section.